x-pack/plugins/alerts/server/alerts_client.ts

```diff
@@ -228,6 +229,11 @@ export class AlertsClient {
       params: validatedAlertTypeParams as RawAlert['params'],
       muteAll: false,
       mutedInstanceIds: [],
+      executionStatus: {
+        status: 'unknown',
```

Comment:
I kinda look at the "set of these strings" type as being the ceremony-free version of string enums.  Is there an explicit value in using enums instead?  The upside to not using them is not having to maintain the duplication of the enum keys / values, and not having to have access to that enum type in code that needs it.  ie, less ceremony :-)

Would the enum type be exposed in the Alert objects themselves, or just an internal detail?  I _think_ we could type it in the Alert and RawAlert as enum safely, but not quite sure.  Another reason I generally avoid enums, because I always have to go read the chapter on them in the TS handbook :-)